language/move-compiler/tests/move_check/liveness/unused_assignment.exp

```diff
@@ -1,54 +1,66 @@
+warning[W09002]: unused variable
```

Comment:
The errors are _slightly_ different. It is an unused assignment vs an unused declaration. I've suppressed the error for this case since it will get an unused assignment error.